tests/karma/single.test.path.js

```diff
@@ -1,6 +1,6 @@
 /*global testPaths:true */
 testPaths = [
-	"profile/wearable/widget/wearable/AssistPanel"
+	"core/widget/core/Appbar"
```

Comment:
one question,
will you test other widgets with karma helper too?

Reply:
Yes.
 This is configuration only for AppBar - usefully for check this PR
`> npm run test:karma-single-test`

All test are defined in `testPaths.js`
`> npm run test:karma`